### PR TITLE
Site settings

### PR DESCRIPTION
**Pull request by ehlertjd**
Adds `/api/site/settings` endpoint which allows setting/retrieving the `center_gears` property mentioned in #1475 

### Review Checklist

- Tests were added to cover all code changes
- Documentation was added / updated
- Code and tests follow standards in CONTRIBUTING.md

### CODE REVIEW

**Comment by nagem** [comment]
I wanted to create a list of all site-wide settings:
  - Center/Lab pays for billing
  - Gear Rules
  - Context Values (not implemented)
 
If I'm being loose with the word "settings":
  - Installed Gears
  - Modalities
  - Devices
  - Users?
  - Groups??
 
Is there anything I'm missing? I was working with @maxbailey on a few UI design changes to incorporate the site settings additions.

**Comment by ehlertjd** [comment]
There are site-level data-views, but I think that's a whole other discussion.
Otherwise that list looks current.

File: swagger/schemas/definitions/config.json [hunk 1]
```diff
@@ -77,6 +77,28 @@
...
             ]
+        },
+
+        "site-settings": {
+            "type": "object",
+            "properties": {
+                "center_gears": {
+                    "type": "array",
+                    "items": {"type": "string"},
+                    "description": "A list of gears that are treated as center-pays for the purpose of the billing report"
```
Comment (nagem) [inline swagger/schemas/definitions/config.json]: Is this a gear `alg` name or a string of the id of a specific gear?

Comment (nagem) [reply swagger/schemas/definitions/config.json]: I saw in an example below it's `alg`. Maybe add that to the description?

**Pull request closed without merging**